1.segment-data/README.md

```diff
@@ -0,0 +1,55 @@
+# Segment Cell Painting Images
+
+In this module, we present our pipeline for segmenting nuclei and cyoplasm from the Cell Health data.
+
+### Segmentation
+
+We use the CellPose segmentation algorithim to segment the nuclei from each mitosis movie. 
+CellPose was first introduced in [Stringer, C., Wang, T., Michaelos, M. et al., 2020](https://doi.org/10.1038/s41592-020-01018-x) and we use the [python implementation](https://github.com/mouseland/cellpose).
+
+Stringer et al. trained the CellPose segmentation models on a diverse set of cell images and is therefore a good selection for our use case.
+The CellPose python implementation was particularly useful for building reproducible pipelines.
+
+After manually experiementing with CellPose on about 10 Cell Health nuclei images (various cell lines), we settled on the following parameters for CellPose nuclei segmentation:
+- `model_type = "cyto"` This parameter forces CellPose to use the cytoplasm model, which we found segments nuclei in the Cell Health data significantly better than the nucleus model. 
+More information about CellPose models can be found at https://cellpose.readthedocs.io/en/latest/models.html.
+- `channels = [0,0]` This parameter forces the model to segment cells in grayscale (in the case of Cell Health data single channel images).
+- `diameter = 80` This parameter indicates to the model that the average cell diameter is 80 pixels.
```

Comment:
Is this the same for all cell lines? IIRC, ES2 cells are larger. (but mayeb this parameter doesn't matter too much)

Reply:
Yes, these are the same for all cell lines. We manually testing these configurations for all 3 cell lines. This parameter doesn't matter too much, mostly just ensures the cells don't get over-segmented (produce very small nuclei segmentation).